src/modules/app/VirtualMouseModule.js

```diff
@@ -48,7 +51,17 @@ export class VirtualMouseModule extends Events {
       'mousemove'
     ].forEach(ev => this.on(ev, e => self.emit(ev, e)));
 
-    this.on('mousemove', e => self.update(e));
+    let cacheX = 0;
+    let cacheY = 0;
+
+    this.on('mousemove', e => {
+      if (document.pointerLockElement !== null) {
+        cacheX += e.movementX;
+        cacheY += e.movementY;
+
+        self.update(e, cacheX, cacheY);
+      } else self.update(e);
```

Comment:
I also prefer braces everywhere, but our standard is none if single instruction..